### ISSUE TITLE
Invalid creative error message

### ISSUE THREAD

**Issue by HerminaC** [oyiptong/splice#262]
This is more a nice to have. When uploading an image  for a tile, if the image doesn't comply with the format/dimension requirements, the user will get an "invalid creative" error message without any details. 
It would be good to include in the message the acceptable format or dimensions so that she/he knows what action to take.
![invalid creative](https://cloud.githubusercontent.com/assets/7406447/11396792/3d0efd5c-936c-11e5-88aa-ef0bda9456ff.png)

**Comment by tkiethanom** [comment]
@k88hudson @ncloudioj I believe this error is coming from the API.

**Comment by ncloudioj** [comment]
@tkiethanom True, that message was from the server. We'll do the server side data validation for the file extension (png, jpg, svg) and the actual image (for example, a file with valid file extension but not actually in that format).

Could be helpful if the UI also checks the file extension before uploading it to the server.

**Comment by tkiethanom** [comment]
@ncloudioj Will do! I think it's important that the error message clearly states what the problem is too.

**Comment by ncloudioj** [comment]
@tkiethanom Yup, definitely will revisit the error message again.
